apps/disease_control/models.py

```diff
@@ -63,8 +64,17 @@ def get_context(self, request):
 
 class DiseasePage(Page):
     parent_page_types = ["disease_control.DiseasesAndConditionsPage"]
-
-    description = RichTextField(blank=True)
+    subpage_types = []
+
+    at_a_glance = RichTextField(blank=True)
```

Comment:
Oops! I totally missed that. I have now brought it back. But I've altered the migration (rather than adding a second migration), so you will have to back out your migration locally before checking again.